### PR TITLE
Update SDK to run on Open-JDK 8

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**
This is an intermediate step in the migration towards OpenJDK 11.

This PR only replaces Oracle JRE 8 with OpenJDK 8.

### CODE REVIEW

File: tools/universe/package_builder.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
-_jre_url = "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz"
+_jre_url = "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u212-b03/OpenJDK8U-jdk_x64_linux_hotspot_8u212b03.tar.gz"
```
Comment (VladimirKravtsov) [inline tools/universe/package_builder.py]: Hi dear @kaiwalyajoshi, @samvantran,
I just noticed that previously **JRE** was used here and in other places, but now it's pointing to **JDK**, although I guess make sense switch it to **JRE**.  You can find it here https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u212-b03/OpenJDK8U-jre_x64_linux_hotspot_8u212b03.tar.gz

Comment (kaiwalyajoshi) [reply tools/universe/package_builder.py]: We're using `https://downloads.mesosphere.com/java/openjdk-8u212b03-hotspot-linux-x64.tar.gz` to maintain consistency with the other named packages.

The two images are exactly the same, just named differently.

File: tools/universe/package_builder.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
-_jre_url = "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz"
+_jre_url = "https://downloads.mesosphere.com/java/openjdk-8u212b03-hotspot-linux-x64.tar.gz"
```
Comment (takirala) [inline tools/universe/package_builder.py]: Wouldn't jre's at https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=hotspot be sufficient ?

Comment (kaiwalyajoshi) [reply tools/universe/package_builder.py]: Good catch! Updated to fetch JRE here.

File: tools/universe/package_builder.py (hunk 1 above)
Comment (VladimirKravtsov) [reply tools/universe/package_builder.py]: After editing your the message, LGTM:-)

**Comment by kaiwalyajoshi** [comment]
Replaced by this PR: https://github.com/mesosphere/dcos-commons/pull/3060

**Pull request closed without merging**